mlflow/server/handlers.py

```diff
@@ -495,6 +506,20 @@ def wrapper(*args, **kwargs):
 ]
 
 
+def _guess_mime_type(filename):
+    extension = os.path.splitext(filename)[-1].replace(".", "")
+    # for MLmodel/mlproject with no extensions
+    if extension == "":
+        extension = filename
```

Comment:
is there something odd happening with windows on this logic assignment where it's not recognizing the empty extension type?